docs/addons/instana-addon.md

```diff
@@ -0,0 +1,199 @@
+
+## Prerequisites
+
+Ensure that you have installed the following tools on your machine.
+
+1. [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+2. [kubectl](https://Kubernetes.io/docs/tasks/tools/)
+3. [cdk](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_install)
+4. [npm](https://docs.npmjs.com/cli/v8/commands/npm-install)
+5. Instana backend application - Use SaaS (eg [aws](https://aws.amazon.com/marketplace/pp/prodview-hnqy5e3t3fzda?sr=0-1&ref_=beagle&applicationId=AWSMPContessa)) or Install self-hosted Instana backend ([on-premises](https://www.ibm.com/docs/en/instana-observability/current?topic=installing-configuring-self-hosted-instana-backend-premises))
+
+## Installation
+
+To create a new project and install dependencies, follow these steps from Amazon EKS Blueprints [Quick Start](https://aws-quickstart.github.io/cdk-eks-blueprints/getting-started)
+
+Use following command to Install IBM Instana Addon:
+
+```shell
+npm i @instana/aws-eks-blueprint-addon
+```
+
+## Instana Agent Configuration
+Go to your Instana installation (Instana User Interface), click ... More > Agents > Installing Instana Agents and select 'Kubernetes' platform to get the Instana Agent Key, Instana Service Endpoint, Instana Service port. These steps are also described [here](https://www.ibm.com/docs/en/instana-observability/218?topic=instana-endpoints-keys) or in the screenshot below.
+
+![Instana Agent Configuration](../assets/images/instana-agent.png)
+
+## AWS Secret Manager Secrets (Optional)
+If you wish to use AWS Secret Manager Secrets to pass Instana props (key, endpoint, and port), then you will be required to setup Secrets first.
+
+```shell
+export SECRET_NAME=<aws_secret_name>
+export INSTANA_AGENT_KEY=<instana_key>
+export INSTANA_ENDPOINT_HOST_URL=<instana_host_endpoint>
+export INSTANA_ENDPOINT_HOST_PORT=<instana_port>"
+aws secretsmanager create-secret \
+  --name $SECRET_NAME \
+  --secret-string "{\"INSTANA_AGENT_KEY\":\"${INSTANA_AGENT_KEY}\",
+    \"INSTANA_ENDPOINT_HOST_URL\":\"${INSTANA_ENDPOINT_HOST_URL}\",
+    \"INSTANA_ENDPOINT_HOST_PORT\":\"${INSTANA_ENDPOINT_HOST_PORT}\"
+   }"
+```
+secret_name = AWS Secret Manager Secret name (eg. *instana-secret-params*).
+
+## Usage : Using AWS Secret Manager Secrets 
+To use AWS Secret Manager Secrets follow these steps:
+
+1. The actual settings for the secret name (```secretParamName```) are expected to be specified in the CDK context. Generically it is inside the cdk.context.json file of the current directory or in `~/.cdk.json` in your home directory.
+
+	 Example settings: Update the context in `cdk.json` file located in `cdk-eks-blueprints-patterns` directory
+	 ```json
+	"context": {
+         "secretParamName": "instana-secret-params"
+     }
+    ```
+
+2. Go to project/bin/main.ts
```

Comment:
there is no bin/main.ts in this repo.

Reply:
Done. changed to bin/<your-project-name>.ts

Reply:
Since this documentation starts with new repo (Quick Start), so i think `bin/<your-project-name>.ts` will be fine. Please let me know if you think otherwise?